File: program/print-reverse-of-a-number/print_reverse_of_a_number.go
package main

import "fmt"

func reverseNumber(num int) int {

	res := 0
	for num > 0 {
		remainder := num % 10
		res = (res * 10) + remainder
		num /= 10
	}
	return res
}

func main() {
	var reverseNo int
	fmt.Print("Enter any Number to find its Reverse = ")
	fmt.Scanln(&reverseNo)

	fmt.Println(reverseNumber(reverseNo))
}

File: program/print-reverse-of-a-number/print_reverse_of_a_number.php
<?php
	$num = readline('Enter a number: ');
	$x = 0;
  $n =$num;
   
  while(floor($num))
  {  
    $mod = $num%10;
    $x = $x * 10 + $mod;  
    $num = $num/10;
  }  
  echo "Reverse of $n is $x.";
?>
